packages/developer-console-spa/src/services/lighthouse.ts

```diff
@@ -9,14 +9,14 @@ import { LHSpaConfigByAppId } from '../utils/gql-queries/lh-spa-config-by-appId'
 export const getLHProjects = async () => {
     return gqlClient({
         query: listLHProjects,
-        variables: {}
+        variables: { limit: 9999 }
```

Comment:
What is the need of enforcing a limit

Reply:
@akhilmhdh Does it take limit by default if we don't pass it explicitly?

Reply:
Yes it has a limit of 1000 i guess so yah you can enforce it like this. 

@riginoommen Its kinda good thing if the list goes more than this obviously we need a pagination as it will take a good amount of load entire project list